src/crypto.js

```diff
@@ -62,14 +61,13 @@ export const authorization =
 
     const signingFunction = async data => ({
       keyId,
-      addr: addr,
+      addr,
       signature: signWithKey(privateKey, data.message, hashAlgorithm),
     })
 
     return {
       ...account,
-      tempId: `${addr}-${keyId}`,
-      addr: fcl.sansPrefix(addr),
+      addr,
```

Comment:
You removed `sansPrefix` here...verifying that's what you want?

Reply:
Yup, awaiting FCL to fix.  The `sansPrefix` shouldn't even be a requirement of the authz function so this sansPrefix is useless (once this is fixed)